velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -580,6 +580,44 @@ struct DateTruncFunction {
   const tz::TimeZone* timeZone_ = nullptr;
 };
 
+/// Truncates a date to a specified time unit. Return NULL if the format is
+/// invalid. Format as abbreviated unit string is allowed.
+template <typename T>
+struct TruncFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void initialize(
+      const std::vector<TypePtr>& /*inputTypes*/,
+      const core::QueryConfig& config,
+      const arg_type<Date>* /*date*/,
+      const arg_type<Varchar>* /*format*/) {
+    timeZone_ = getTimeZoneFromConfig(config);
```

Comment:
Where is the timeZone_ usage?

Reply:
It's redundant, I'll remove it right away, thanks.